qfieldsync/ui/package_dialog.ui

```diff
@@ -111,11 +81,33 @@
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
-          <string>Export Directory</string>
+          <string>Packaged Project Title</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
-           <widget class="QLineEdit" name="manualDir"/>
+           <widget class="QLineEdit" name="packaged_project_title"/>
+          </item>
+         </layout>
+        </widget>
+       </item>
+       <item>
+         <widget class="QGroupBox" name="groupBox">
+         <property name="title">
+          <string>Packaged Project Filename</string>
+         </property>
+         <layout class="QGridLayout" name="gridLayout">
+       <item>
+         <widget class="QLineEdit" name="manualDir">
+           <property name="maximumSize">
+             <size>
+             <width>0</width>
+             <height>0</height>
+             </size>
+           </property>
+         </widget>
+       </item>
```

Comment:
Why do we have the `manualDir` here? Also I think there is some kind of an error with the XML, or I miss somehow the closing `</item>` for line 93. How did you edit this?